app/models/specialist_document.rb

```diff
@@ -86,7 +86,11 @@ def update(params)
   def published?
     editions.any?(&:published?)
   end
-  alias_method :has_ever_been_published?, :published?
+
+  def has_ever_been_published?
+    return false if @editions.size == 1 && needs_exporting?
+    published?
```

Comment:
I'm definitely missing context, but looking at this, I'm unsure why its mixing in exporting and publishing? I'm guessing that publishing can be interpreted in the context of the Publishing API, but I'm less sure what exporting is?